settings.json

```diff
@@ -12,7 +12,7 @@
   "log-folder": "logs/",
   "fluid-compiler-path": "./",
   "fluid-temp-folder": "./fluid-temp",
-  "agent-limit": 3,
+  "agent-limit": 0,
```

Comment:
Do you need to reset this back to 3? (I can't remember what `agent-limit` actually controls..)

Reply:
The number of times that the agent try to generate the expression after an error.